infobserve/processors/yara_processor.py

```diff
@@ -53,45 +50,24 @@ async def process(self):
         The consumer function for the source queue.
         Removes Sources from the Processing Queue, tries to match them
         against the provided Yara rules. Places any matches into the Database
-        Queue for further processing and storage (TBI - Right now, it simply
-        logs matches)
+        Queue(_db_queue)
```

Comment:
I don't think the name of the variable it is placed in is important here (:scissors: `(_db_queue)`)